python_ta/transforms/type_inference_visitor.py

```diff
@@ -523,7 +523,7 @@ def visit_classdef(self, node: astroid.ClassDef) -> None:
         self.type_constraints.unify(self.lookup_inf_type(node.parent, node.name),
                                     Type[_ForwardRef(node.name)], node)
 
-        self.type_store.classes[node.name]['__bases'] = [self.type_store.node_to_type(base)
+        self.type_store.classes[node.name]['__bases'] = [_node_to_type(base)
```

Comment:
It looks like this function has already been implemented in base.py. Can't believe I missed it.